pkg/manager/member/tidb_init_manager.go

```diff
@@ -347,10 +347,13 @@ func (m *tidbInitManager) makeTiDBInitJob(ti *v1alpha1.TidbInitializer) (*batchv
 	}
 
 	meta, initLabel := getInitMeta(ti)
+	podLabels := util.CombineStringMap(initLabel, ti.Spec.Labels)
+	podAnnotations := util.CopyStringMap(ti.Spec.Annotations)
 
 	podSpec := &corev1.PodTemplateSpec{
 		ObjectMeta: metav1.ObjectMeta{
-			Labels: initLabel.Labels(),
+			Labels:      podLabels,
+			Annotations: podAnnotations,
```

Comment:
Is it OK to use `meta.Annotations` here?

Reply:
YES, https://github.com/pingcap/tidb-operator/pull/4029/commits/3ea7b432dc9678acd1f63de8d00b47553b4ddf7a